src/phoenix/experimental/evals/models/bedrock.py

```diff
@@ -106,6 +106,9 @@ def get_tokens_from_text(self, text: str) -> List[int]:
     def get_text_from_tokens(self, tokens: List[int]) -> str:
         return self.encoder.decode(tokens)
 
+    async def _async_generate(self, prompt: str, **kwargs: Dict[str, Any]) -> str:
+        return self._generate(prompt, **kwargs)
```

Comment:
clarrification: Is this correct or a step gap?

Reply:
Yeah, all the non-openai models will not run asynchronously until I carve out time to figure out their async bindings

Reply:
sg